<?xml version="1.0" ?>
<launch>

 <!-- Turtlesim Node-->
  <!-- <node pkg="turtlesim" type="turtlesim_node" name="sim"/> -->


 <!-- joy node -->
  <node respawn="true" pkg="joy" type="joy_node" name="ddr_joy" >
    <param name="dev" type="string" value="/dev/input/js1" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
  <param name="axis_linear" value="1" type="int"/>
  <param name="axis_angular" value="3" type="int"/>
  <param name="scale_linear" value="0.5" type="double"/>
  <param name="scale_angular" value="2" type="double"/>

  <node pkg="ddr_pkg" type="ddr_teleop_twist_joy" name="ddr_teleop_twist_joy"/>

</launch>
